src/Sulu/Bundle/WebsiteBundle/Sitemap/SitemapDumper.php

```diff
@@ -0,0 +1,105 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\WebsiteBundle\Sitemap;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Templating\EngineInterface;
+
+/**
+ * Class SitemapDumper
+ */
```

Comment:
Please add better PHPDocs, also to all the methods (description of the method, types and description for parameters)